web/docs/try-vast/quickstart.md

```diff
@@ -0,0 +1,84 @@
+# Quickstart
```

Comment:
This file should not be in version control, should it?

Reply:
Correct, will remove it.